Makefile.toml

```diff
@@ -69,6 +69,47 @@ end
 echo "License header check complete"
 '''
 
+[tasks.generate-readmes]
```

Comment:
Suggestion(blocking): ensure Duckscript script return non-zero return code on error

I found that without also adding the directive as on line 34 (`exit_on_error true`), you can have a Duckscript script trigger an error and terminate (via `trigger_error`), yet return a 0 status code (success) to the shell. This matters because the failure then is ultimately not propagated to Github Actions as intended, so errors become silent (thus, fall through the cracks).

Can you see what the failure mode looks like when running `cargo generate-readmes` followed by `echo $?` to get the return code?

If you test that out and find that the directive isn't necessary, then there are no changes to make.

Reply:
Good catch!